client/src/components/Faqs.component.tsx

```diff
@@ -0,0 +1,36 @@
+import { IoIosArrowDown } from "react-icons/io";
+import { FaqConfig } from "../config/FaqConfig";
+
+export default function Faqs() {
+  return (
+    <>
+      <div className="flex justify-center">
+        <h1 className="my-10 text-4xl font-bold">Întrebări Frecvente</h1>
+      </div>
+      <div className="flex justify-center ">
+        <div className="lg:w-1/2 md:w-2/3 mx-5 md:mx-0 shadow-2xl ">
+          {FaqConfig.map((question, index) => (
+            <div
+              className="group flex flex-col border-b border-black border-opacity-15 bg-white bg-grey-100 "
+              tabIndex={index}
+              key={index}
+            >
+              <div className="flex cursor-pointer items-center bg-stone-100 justify-between p-4 duration-500 hover:bg-[#aff1f3]">
+                <span className=" text-xl text-gray-600 group-hover:translate-x-2 duration-500 ">
+                  {question.question}
+                </span>
+                <IoIosArrowDown
+                  size={20}
+                  className="transition-all duration-500 group-focus:-rotate-180"
+                />
+              </div>
```

Comment:
Please respect the previous design. Please include the rest of the items as well. Essentially, the question number and those red dots. In the case of mapping, the object number helps as an index.